src/utils/TedexSetup.ts

```diff
@@ -0,0 +1,80 @@
+import { TraderClient, TradeType } from 'tdex-sdk';
+import { toSatoshi, fromSatoshi } from './format';
+import BigNumber from 'bignumber.js';
+import { Market, Markets } from './constants';
+
+interface KnownAsset {
+  title: string;
+  image: string;
+  hash: string;
+  precision: number;
+}
+
+interface PairMarket {
```

Comment:
I think the object with a market as key is useless. 

Also lets use camelCase to be consistent with tdex sdk

```suggestion
interface PairMarket {
    baseAsset: string;
    quoteAsset: string;
  }
}
```

Reply:
fixed this